ERCS/erc-mtc-zk.md

```diff
@@ -0,0 +1,143 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 8036
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.